gen/gen.go

```diff
@@ -176,10 +177,19 @@ fragment TypeRef on __Type {
   }
 }
 `
+	apiEndpoint := "https://" + apiHost + "/query"
```

Comment:
we should define apiHost := "us.api.mondoo.com" close to this code since its otherwise difficult to read

Reply:
we should parse the url and prepend https if no schema is defined

Reply:
Fixed.